docs/api/covidcast-signals/nchs-mortality.md

```diff
@@ -0,0 +1,56 @@
+available by National Center for Health statistics [(NCHS)](https://www.cdc.gov/nchs/nvss/vsrr/COVID19/index.htm).
+
+| Signal | Description |
+| --- | --- |
+| `covid_deaths_num` | Number of all the weekly new deaths with confirmed or presumed COVID-19 |
+| `covid_deaths_prop` | Number of all the weekly new deaths with confirmed or presumed COVID-19 per 100,000 population |
+| `total_deaths_num` | Number of weekly new deaths from all causes |
+| `total_deaths_prop` | Number of weekly new deaths from all causes per 100,000 population |
+| `influenza_deaths_num` | Number of weekly new deaths involving Influenza |
+| `influenza_deaths_prop` | Number of weekly new deaths involving Influenza per 100,000 population |
+| `pneumonia_deaths_num` | Number of weekly new deaths involving Pneumonia, excluding Influenza deaths |
+| `pneumonia_deaths_prop` | Number of weekly new deaths involving Pneumonia, excluding Influenza deaths per 100,000 population |
+| `pneumonia_and_covid_deaths_num`| Number of weekly new deaths involving COVID-19 and Pneumonia, excluding Influenza |
+|`pneumonia_and_covid_deaths_prop`| Number of weekly new deaths involving COVID-19 and Pneumonia, excluding Influenza per 100,000 population |
+|`pneumonia_influenza_or_covid_19_deaths_num`| Number of weekly new deaths involving Pneumonia, Influenza, or COVID-19|
+|`pneumonia_influenza_or_covid_19_deaths_prop`| Number of weekly new deaths involving Pneumonia, Influenza, or COVID-19 per 100,000 population|
+|`percent_of_expected_deaths`| The number of weekly new deaths for all causes in 2020 compared to the average number across the same week in 2017–2019|
+
+These signals are taken directly from [Table 1](https://www.cdc.gov/nchs/nvss/vsrr/COVID19/index.htm) without changes. National provisional counts include deaths occurring within the 50 states and the District of Columbia that have been received and coded as of the date specified. The deaths are classified based on a new ICD-10 code. The codes that are considered for each signals are described in details [here](https://github.com/cmu-delphi/covidcast-indicators/blob/nchs_mortality/nchs_mortality/DETAILS.md#metrics-level-1-m1). We export the state-level data as-is in a weekly format. 
+
+## Table of contents
+{: .no_toc .text-delta}
+
+1. TOC
+{:toc}
+
+## Geographical Exceptions
+
+New York City is considered as a special state in the NCHS Mortality data, but we don't consider NYC separately. The death counts for NYC would be included in New York State in our reports.
+
+## Report Using Epiweeks
+
+We report the NCHS Mortality data in a weekly format (weekly_YYYYWW, where YYYYWW refers to an epiweek). However, NCHS reports their weekly data from Saturday to Saturday. We assume there is a one day shift. For example, they report a death counts for Alaska in a week starting from date D, we will report the timestamp of this report as the corresponding epiweek of date(D + 1).
```

Comment:
So when someone requests this data, what is the `time_value`? I don't think we have any weekly signals yet, so it may be worth being explicit about how to interpret the `time_value` from the API.

Reply:
If the time_value in API is based on the file names, so they should be epiweeks. Not quite sure about this, need @krivard's answer.

Reply:
`time_type=week&time_value={YYYYWW}`

[Canonical definition of CDC epiweeks](https://www.cmmcp.org/mosquito-surveillance-data/pages/epi-week-calendars-2008-2020#:~:text=WHAT%20IS%20AN%20%22EPI%20WEEK,of%20data%20year%20after%20year.) (I know it's a mosquito surveillance project but that seems to be the one everybody refers back to so 🤷 )

[Python library for converting between dates and epiweeks](https://pypi.org/project/epiweeks/); eg

```
from datetime import date
import epiweeks
print(epiweeks.Week.fromdate(date.today()))
```